android/filament-android/src/main/java/com/google/android/filament/RenderableManager.java

```diff
@@ -821,6 +816,7 @@ public long getNativeObject() {
     private static native int nSetBonesAsQuaternions(long nativeObject, int i, Buffer quaternions, int remaining, int boneCount, int offset);
     private static native void nSetMorphWeights(long nativeObject, int instance, float[] weights, int offset);
     private static native void nSetMorphTargetBufferAt(long nativeObject, int i, int level, int primitiveIndex, long nativeMorphTargetBuffer, int offset, int count);
+    private static native void nSetMorphTargetBufferAt(long nativeObject, int i, int level, int primitiveIndex, long nativeMorphTargetBuffer);
```

Comment:
Just implement this one in Java, by calling the other one and your new `getMorphTargetCount()`. It's just easier. In the javadoc just say it's a helper.

Reply:
Ok it's done. nit: Are you saying new `SetMorphTargetBufferAt()` instead of `getMorphTargetCount()` right?